helm/promxy-app/values.yaml

```diff
@@ -1,7 +1,9 @@
-Installation:
-  V1:
-    Registry:
-      Domain: quay.io
+monitoring:
+  prometheus:
+    host: ""
+    letsencrypt: true
```

Comment:
@giantswarm/team-atlas Is it ok if `letsencrypt` is set to `true` by default? I'm thinking about WC deployment.

Reply:
Yes